src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,136 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.22.0.dev2 (10/13/2019)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+* Deprecate mypy-version flag and add new version flag (#8463)
+  `Issue #7627 <https://github.com/pantsbuild/pants/issues/7627>`_
+  `PR #8463 <https://github.com/pantsbuild/pants/pull/8463>`_
+
+* Fix tensorflow==1.14 globbing with platform-specific native_lib_names (#8378)
+  `PR #8378 <https://github.com/pantsbuild/pants/pull/8378>`_
+
+* Error out if the user explicitly tries to test a non-test tgt. (#8402)
+  `PR #8402 <https://github.com/pantsbuild/pants/pull/8402>`_
+
+* Revert "Format our python code with Black (#8391)" (#8409)
+  `PR #8391 <https://github.com/pantsbuild/pants/pull/8391>`_
+  `PR #8409 <https://github.com/pantsbuild/pants/pull/8409>`_
+
+New Features
+~~~~~~~~~~~~
+* Allow ignoring flake errors by name in addition to code. (#8444)
+  `PR #8444 <https://github.com/pantsbuild/pants/pull/8444>`_
+
+* Allow extra remote execution headers to be set (#8430)
+  `PR #8430 <https://github.com/pantsbuild/pants/pull/8430>`_
+
+* Start creating `binary` goal with support for python (#8399)
+  `PR #8399 <https://github.com/pantsbuild/pants/pull/8399>`_
+
+* Add support for running Pants with Python 3.8 (#8415)
+  `PR #8415 <https://github.com/pantsbuild/pants/pull/8415>`_
+
+Bugfixes
+~~~~~~~~
+* Send cumulative_timings & recorded_options in stats v2 too. (#8456)
+  `PR #8456 <https://github.com/pantsbuild/pants/pull/8456>`_
+
+* remove @_algebraic_data decorator to support @dataclass and mypy (#8437)
```

Comment:
Refactoring

Reply:
Is the reference to mypy choking just on the Pants codebase?

Reply:
Yes, MyPy running on the pants codebase itself